app/src/main/java/org/mozilla/fenix/wallpapers/WallpaperState.kt

```diff
@@ -20,4 +25,44 @@ data class WallpaperState(
             availableWallpapers = listOf(),
         )
     }
+
+    /**
+     * Helper property used to obtain the [Color] to fill-in cards in front of a wallpaper.
+     *
+     * @return The appropriate light or dark wallpaper card [Color], if available, otherwise a default.
+     */
+    val wallpaperCardColor: Color
+        @Composable get() = when {
+            currentWallpaper.cardColorLight != null && currentWallpaper.cardColorDark != null -> {
+                if (isSystemInDarkTheme()) {
+                    Color(currentWallpaper.cardColorDark)
+                } else {
+                    Color(currentWallpaper.cardColorLight)
+                }
+            }
+            else -> FirefoxTheme.colors.layer2
+        }
+
+    /**
+     * Run the Composable [run] block only if the current wallpaper's card colors are available.
+     */
+    @Composable
+    fun composeRunIfWallpaperCardColorsAreAvailable(
```

Comment:
Is the `compose` naming here necessary? I'd hope the compiler was smart enough to do the right thing given the annotation and composable parameter

Reply:
I was getting a platform declaration error:

<img width="910" alt="image" src="https://user-images.githubusercontent.com/87384386/197899131-641934f9-1fe4-4e1a-8e22-f3e2f25ad952.png">

Reply:
Ah, that's too bad.

Reply:
it will at least, hopefully, be "temporary", since it's only used with top sites, as that's the last View-based component of homepage.